src/handle_finalize.c

```diff
@@ -8,9 +8,21 @@ void handle_finalize(void *parameters) {
         case PURCHASE:
             msg->numScreens = 1;
             break;
+        case PURCHASE_HOLDER:
```

Comment:
@jakerockland We now display the additional screens 🥳

Reply:
BASED 🙏